src/translations/en.json

```diff
@@ -677,6 +677,14 @@
               "unsupported_platform": "Unsupported platform: {platform}",
               "type_select": "Trigger type",
               "type": {
+                "device": {
+                  "label": "Device",
+                  "trigger_type": {
+                    "turn_on": "{entity_id} turned on",
```

Comment:
Since triggers can be defined in many places, each integration should have their own place where we retrieve translations from.

`device_automation.light.trigger.turn_on` etc

Reply:
Maybe it should be: `device_automation.toggle.trigger.turn_on` or `device_automation.toggle_entity.trigger.turn_on` instead?

Reply:
Fixed; translations moved to backend.